src/OrchardCore/OrchardCore.Media.Abstractions/MediaFileStore.cs

```diff
@@ -1,25 +1,28 @@
-using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Threading.Tasks;
 using OrchardCore.FileStorage;
 
-namespace OrchardCore.Media.Services
+namespace OrchardCore.Media
 {
     public class MediaFileStore : IMediaFileStore
     {
```

Comment:
Moved the MediaFileStore implementation to the abstractions project, as it is basically an abstraction, and meant that the Media.Azure module doesn't need to depend on the Media Module

Reply:
Done. The MediaFileStore and Cache Provider moved to a new `OrchardCore.Media.Core` project as default implementations, that any module could reuse.